diff --git a/src/open_llm_vtuber/tts/tts_factory.py b/src/open_llm_vtuber/tts/tts_factory.py
index 0804c46..9c24f5e 100644
--- a/src/open_llm_vtuber/tts/tts_factory.py
+++ b/src/open_llm_vtuber/tts/tts_factory.py
@@ -23,6 +23,13 @@ class TTSFactory:
             from .edge_tts import TTSEngine as EdgeTTSEngine
 
             return EdgeTTSEngine(kwargs.get("voice"))
+        elif engine_type == "voicevox_tts":
+            from .voicevox_tts import TTSEngine as VoicevoxTTSEngine
+
+            return VoicevoxTTSEngine(
+                base_url=kwargs.get("base_url"),
+                speaker_id=kwargs.get("speaker_id"),
+            )
         elif engine_type == "pyttsx3_tts":
             from .pyttsx3_tts import TTSEngine as Pyttsx3TTSEngine
 
